srt-protocol/src/protocol/sender/congestion_control.rs

```diff
@@ -40,40 +40,92 @@ impl StatsWindow<MessageStats> {
 // rate in bytes per second
 type DataRate = u64;
 
-// TODO: move data rate algorithm configuration to a public protocol configuration module
-//       for now, just ignore that it's never used
-#[allow(dead_code)]
-#[derive(Debug)]
-pub(crate) enum LiveDataRate {
-    Fixed {
+// https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-00#section-5.1.1
+//
+/// Note that Maximum Bandwidth, Input Rate, and Input Rate Estimate are bytes per second
+/// and Overhead is a percentage.
+#[derive(Debug, Clone)]
+pub enum LiveBandwidthMode {
+    /// Set the maximum bandwidth explicitly.
+    ///
+    /// The recommended default value is 1 Gbps. The default value is set only for live streaming.
+    ///
+    /// Note that this static setting is not well-suited to a variable input, like when you change the bitrate on an encoder.
+    /// Each time the input bitrate is configured on the encoder, this value should also be reconfigured.
+    Set(DataRate), // m_llMaxBW != 0
+
+    /// Set the SRT send input rate and overhead.
+    /// In this mode, SRT calculates the maximum bandwidth as follows:
+    ///
+    ///   Maximum Bandwidth = Input Rate * (1 + Overhead / 100)
```

Comment:
This implies overhead is % (as it is in the reference impl), but below it's marked as a DataRate in bytes per second?